docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2.1'
+services:
+  postgres:
+    image: postgres:9.3
+
+  conjur:
+    image: registry.tld/possum:0.1.0-stable
```

Comment:
This is the only remaining reference to `possum` in this project. I believe the `conjur` image is in the registry now so should we switch over to that?